ubuntu.md

```diff
@@ -0,0 +1,31 @@
+### Ubuntu 16.04
+Instructions for installing VirtualBox & Vagrant on Ubuntu 16.04
+
+```shell
+sudo apt update
+sudo apt -y --force-yes upgrade
```

Comment:
saw it, the `--allow*` descriptions and which to choose wasn't easy to find. Please suggest one.